scripts/install/etc/sysconfig/motr

```diff
@@ -63,9 +63,17 @@ MOTR_M0T1FS_MOUNTPOINT='/mnt/m0t1fs'
 MOTR_M0D_MAX_RPC_MSG_SIZE=524288
 
 # Minimum rpc receive queue length.
+# Packets from clients will be dropped if the queue is not big enough,
+# and this might affect the performance dramatically. Increase this
+# (for example, to 512) to avoid packets drops on receive.
 #MOTR_M0D_MIN_RPC_RECVQ_LEN=16
 
-# ioservice buffer pool configuration
+# ioservice buffer pool configuration.
+# This parameter relates to the above MOTR_M0D_MIN_RPC_RECVQ_LEN, it
+# controls the number of spare buffers on receive for 0-copy. Set it
+# to the value which is about twice more than MOTR_M0D_MIN_RPC_RECVQ_LEN.
+# (One RPC msg may have several 0-copy transfers. For example, 2MB
+# unit write request will have two 0-copy , as LNet max frame size is 1MB.)
```

Comment:
Let's not add Lnet, with Libfab there is no restriction of 1MB.

Reply:
Ok, dropped.

Reply:
Ok, dropped.

Reply:
Ok, dropped.